manager/catalog_manager.go

```diff
@@ -109,13 +195,18 @@ func SetEnv() {
 						//lowercase the scheme
 						url = strings.ToLower(tokens[1][:index]) + tokens[1][index:]
 					}
+					if *catalogURLBranch != "" {
+						newCatalog.URLBranch = *catalogURLBranch
+						newCatalog.catalogBranchDir = CatalogRootDir + tokens[0] //+ "/" + *catalogURLBranch
+					}
```

Comment:
Value of 'newCatalog.catalogBranchDir' seems same as 'newCatalog.catalogRoot'
Do we need this separate field?